view_component-form.gemspec

```
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = Gem::Requirement.new(">= 2.7.0")
 
-  spec.add_dependency "actionview", [">= 6.0.0", "< 7.0"]
-  spec.add_dependency "activesupport", [">= 6.0.0", "< 7.0"]
+  spec.add_dependency "actionview", [">= 6.0.0", "< 8.0"]
+  spec.add_dependency "activesupport", [">= 6.0.0", "< 8.0"]
```

```suggestion
  spec.add_dependency "actionview", [">= 6.0.0", "< 7.1"]
  spec.add_dependency "activesupport", [">= 6.0.0", "< 7.1"]
```

Maybe it is better to not allow 7.1 for now